api.ts

```diff
@@ -261,6 +261,8 @@ export type GenericDynamicSyncTable = DynamicSyncTableDef<any, any, ParamDefs, a
 export type SyncTable = GenericSyncTable | GenericDynamicSyncTable;
 
 /**
+ * @deprecated
```

Comment:
How come this one?

Reply:
this is export from `legacy_exports`. maybe it's a mistake?

Reply:
Ah ok. I would just remove `@deprecated` but you can leave otherwise leave it alone for now.

I think what probably happened is that everything that used to be in `index.ts` that we seemed to only use for implementation purposes I moved to legacy_exports. I think I was trying to avoid importing anything that wasn't in index.ts or legacy_exports.ts so we could be disciplined about what the interface of the SDK was. We kind of gave up on that for decent reasons and so if it was today I would just remove this from legacy_exports and import it directly from `/dist/api`, but no need to bother right now.